src/MahApps.Metro/Controls/Dialogs/BaseMetroDialog.cs

```diff
@@ -103,12 +103,13 @@ public double DialogMessageFontSize
         }
 
         /// <summary>Identifies the <see cref="DialogButtonFontSize"/> dependency property.</summary>
-        public static readonly DependencyProperty DialogButtonFontSizeProperty = DependencyProperty.Register(nameof(DialogButtonFontSize), typeof(double), typeof(BaseMetroDialog), new PropertyMetadata(12D));
+        public static readonly DependencyProperty DialogButtonFontSizeProperty = DependencyProperty.Register(nameof(DialogButtonFontSize), typeof(double), typeof(BaseMetroDialog), new PropertyMetadata(SystemFonts.MessageFontSize));
```

Comment:
Heh. Didn't know this existed :D

Reply:
NP, I copied it from the default of the Button control.